packages/tools/src/utilities/planar/filterAnnotationsForDisplay.ts

```diff
@@ -35,6 +35,13 @@ export default function filterAnnotationsForDisplay(
     // composed of multiple unrelated images
     return annotations.filter((annotation) => {
       const imageId = annotation.metadata.referencedImageId;
+
+      if (imageId === undefined) {
```

Comment:
Is there a situation that a StackViewport annotation doesn't have imageId? How is it rendering it if it is undefined?

Reply:
oh I see great